ksqldb-functional-tests/pom.xml

```diff
@@ -156,6 +156,14 @@
       <artifactId>protobuf-java-util</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.errorprone</groupId>
+      <artifactId>error_prone_annotations</artifactId>
+      <version>2.18.0</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Had to add explicitly the dependency, because the "nearest" transitive dependency has a version that it is too old for guava.